src/include/daos_srv/pool.h

```diff
@@ -144,6 +144,8 @@ int ds_pool_svc_destroy(const uuid_t pool_uuid);
 
 int ds_pool_svc_get_acl_prop(uuid_t pool_uuid, d_rank_list_t *ranks,
 			     daos_prop_t **prop);
+int ds_pool_svc_set_prop(uuid_t pool_uuid, d_rank_list_t *ranks,
```

Comment:
Is this only going to be used for a complete override? How do we intend on doing individual ACE updates in the future?

Reply:
In recent discussions we came to the conclusion that three ACL modification commands were needed: overwrite, update, and delete. Overwrite is a complete override. Next in my queue is an update-acl command that will add entries and modify individual existing entries without replacing the whole list.

Reply:
I should add - for the read-modify-write in the update command, I think it needs to happen inside the pool service since we need to do it within a write lock. So there will end up being a `ds_pool_svc_update_acl` function. It can reuse a lot of the underlying logic, but I think it will need its own entry point.